examples/controlpanel/pages/products/index.jsx

```diff
@@ -1,14 +1,19 @@
-import React from 'react';
-import { Container } from 'semantic-ui-react';
-import App from '../../components/App';
-import ProductList from '../../components/products/ProductList';
-import connectApollo from '../../lib/connectApollo';
+import React from "react";
+import { Container } from "semantic-ui-react";
+import { withRouter } from "next/router";
+import App from "../../components/App";
+import ProductList from "../../components/products/ProductList";
+import connectApollo from "../../lib/connectApollo";
 
-export default connectApollo((props) => (
-  <App {...props}>
-    <Container>
-      <h2>Products</h2>
-      <ProductList />
-    </Container>
-  </App>
-));
+export default connectApollo(
+  withRouter(({ router, ...rest }) => {
```

Comment:
Would be better to put `withRouter` inside the compose call of ProductList

Reply:
That should be done, please recheck.